src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -405,7 +408,8 @@ public virtual ClaimsIdentity CreateClaimsIdentity(SecurityToken securityToken,
         /// Gets or sets a delegate that will be used to validate the lifetime of the token
         /// </summary>
         /// <remarks>
-        /// If set, this delegate will be called to validate the lifetime of the token, instead of normal processing. This means that no default lifetime validation will occur.
+        /// If set, this delegate will be called to validate the lifetime of the token, instead of typical processing.
+        /// This means that no default lifetime validation will occur.
```

Comment:
For my own edification, is the update here to avoid long lines you have to scroll to see? If so hooray, I'm very in favor of this. Is that an accessibility improvement or is there another inclusive angle I'm missing?

Reply:
I am changing a word (which is part of the inclusiveness analyzer, removing the word 'normal' for typical/default in this case. Other inclusive terms that could be used are excepted or healthy, depending on the context), and figured since I was editing these lines might as well make them more readable since the line was quite long.

There's no accessibility angle that Inclusive analyzer can give, other than terms that might be exclusive towards people with disabilities.